_posts/2020-11-17-quarkiverse.adoc

```diff
@@ -0,0 +1,75 @@
+---
+layout: post
+title: Quarkiverse Hub - Home of Community Extensions
+date: 2020-11-17
+tags: extensions quarkiverse
+synopsis: Entering a world of extensions
+author: gastaldi
+---
+
+Quarkus is a Kubernetes Native Java stack tailored for the OpenJDK HotSpot and GraalVM, crafted from the best of breed Java libraries and standards, as well as an ever growing ecosystem of its extensions. At the beginning we were accepting all the contributed extensions in the core Quarkus repository. Eventually though it grew too large and lead to a maintenance overhead. In addition to that, in some cases it's simply not always sensible to include an extension into the core Quarkus repository.
+
+Some have since then created their own repositories external to Quarkus, but others just never happened because not everyone wants the overhead of maintaining a build setup. Even the Quarkus core developers sometimes had ideas for an extension but not solid enough to be included in the Quarkus core repository.
+
+Enter the *Quarkiverse*!
+
+image:/assets/images/posts/quarkiverse/quarkiverse.png[]
```

Comment:
that image is too big :) 

Also, I don't see an explanation for what it is - i.e. what isn grok, what is the significance of the arrows etc. 

Thus, definitely make it smaller - and give some context.